scripts/publish.sh

```diff
@@ -34,6 +34,16 @@ if [[ ! "$VERSION" =~ ^([0-9]+\.[0-9]+\.[0-9]+|0\.0\.1-[0-9a-fA-F]{40})$ ]]; the
     exit 1
 fi
 
+# increment stored version
+# NB: macos and linux have different "sed" that don't edit in place the same way
+pushd "$GIT_ROOT_DIR/packages"
+sed -E "s/NANGO_VERSION = '[0-9a-fA-F.-]+/NANGO_VERSION = '$VERSION/" ./shared/lib/version.ts >tmp
+mv tmp ./shared/lib/version.ts
+sed -E "s/NANGO_VERSION = '[0-9a-fA-F.-]+/NANGO_VERSION = '$VERSION/" ./node-client/lib/version.ts >tmp
+mv tmp ./node-client/lib/version.ts
+popd
```

Comment:
This is only run by CI so it doesn't have to be "portable". Otherwisse `awk -i inplace` might be work the same on all OSes